site/_guides/cert-manager.md

```diff
@@ -418,19 +432,23 @@ To request a properly signed certificate from the Let's Encrypt production serve
 
 Create a file called `letsencrypt-prod.yaml` with the following contents:
 
-```
+```yaml
 apiVersion: certmanager.k8s.io/v1alpha1
 kind: ClusterIssuer
 metadata:
   name: letsencrypt-prod
   namespace: cert-manager
+spec:
 spec:
   acme:
     email: user@example.com
-    http01: {}
     privateKeySecretRef:
       name: letsencrypt-prod
     server: https://acme-v02.api.letsencrypt.org/directory
+    solvers:
+    - http01:
+        ingress:
```

Comment:
Is the ingress class needed? What does it default to?

Reply:
There is no default. You must specify this for cert-manager to know the ingress class to put on the Ingress objects it creates. This is new in 0.11.